tools/lib/provision.py

```diff
@@ -391,15 +391,13 @@ def main(options):
     ]
     run_as_root(proxy_env + ["scripts/lib/install-node"], sudo_args = ['-H'])
 
+    if not os.access(NODE_MODULES_CACHE_PATH, os.W_OK):
+        run_as_root(["mkdir", "-p", NODE_MODULES_CACHE_PATH])
+        run_as_root(["chown", "%s:%s" % (user_id, user_id), NODE_MODULES_CACHE_PATH])
```

Comment:
Why do we no longer need to check for a `node_modules` symlink existing?

Reply:
I followed that check back to cd5b38f5d8bdcc1771ad794f37262a61843c56c0, which was a workaround for a bug that hasn’t existed since long before the bionic upgrade.